Source/Relativity.Test.Helpers/Relativity.Test.Helpers/TestHelper.cs

```diff
@@ -10,11 +10,35 @@ namespace Relativity.Test.Helpers
 {
 	public class TestHelper : IHelper
 	{
-        public void Dispose()
+        private readonly string _username;
+        private readonly string _password;
+
+        [Obsolete("Please use the TestHelper.System static method instead")]
+        public TestHelper()

```

Comment:
@ppande1020 are you using the TestHelper constructor, I left it in here for compatibility sake, but wanted to get rid of it. I like the explicit use of the static methods to really show what is going on. If you are not using them I can get rid of this constructor and be a happier camper